### DISCUSSION TITLE
Access Agnostic

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#2] [General]
General Requirement: The API shall be designed and implemented (description and name of parameters) with no reference to a specific Access.

**Comment by bigludo7** [comment]
Yes I think so
From an attribute perspective should something like:
`{
  "phoneNumber": "+346661113334"
}`

With a description:
Subscriber number in E.164 format (starting with country code). Optionally prefixed with '+'. (pattern: ^\+?[0-9]{5,15}$)

**Reply by FabrizioMoggio** [reply]
in the TI API we have:

    PhoneNumber:
      type: string
      pattern: '^\+?[0-9]{5,15}$'
      example: "123456789"
      description: Subscriber number in E.164 format (starting with country code). Optionally prefixed with '+'.


It should be coherent with other CAMARA API. I usually check and update it :-) I will check again.

**Reply by bigludo7** [reply]
Yes I guess this is is the correct definition.

the only thing that could change is having the `+` mandatory (here https://github.com/camaraproject/Commonalities/pull/148) but that's a very small update.

**Comment by FabrizioMoggio** [comment]
thank you, good catch. We will implement it according to that.

**Comment by FabrizioMoggio** [comment]
supported by the current release
